app/javascript/plate-picks/index.js

```diff
@@ -0,0 +1,21 @@
+/* eslint no-console: 0 */
+
+import Vue from 'vue'
+import Vuex from 'vuex'
+import App from './app.vue'
+import storeConfig from './store'
+
+Vue.use(Vuex)
+
+const store = new Vuex.Store({
```

Comment:
It persists until the user navigates away from the page. With single page applications this could be a long time, but in this case any moves away from the page are proper reloads. I haven't looked into the exact details of how to manage vuex lifecyles with SPAs. (A quick google suggests that there may also be issues if we ever enable turbolinks)